tests/test_switch.py

```diff
@@ -198,7 +199,8 @@ async def setup_lights_and_switch(hass, extra_conf=None):
             CONF_SUNSET_TIME: datetime.time(SUNSET.hour),
             CONF_INITIAL_TRANSITION: 0,
             CONF_TRANSITION: 0,
-            CONF_DETECT_NON_HA_CHANGES: True,
+            CONF_DETECT_NON_HA_CHANGES: False,
+            CONF_TAKE_OVER_CONTROL: True,
```

Comment:
Why change these defaults for all tests?

Reply:
I changed them to be certain my test worked as I'm unfamiliar with a lot of the other code.